server/middlewares/validator.js

```diff
@@ -338,4 +343,18 @@ export default class Validator {
         .withMessage('roleId parameter must be an integer'),
     ];
   }
+
+  /**
+   * Validates input data for roles
+   * @static
+   * @returns {array} The array of express validator chains
+   * @memberof Validator
+   */
+  static validateClaps() {
+    return [
+      body('claps')
+        .isInt({ max: CLAPS_LIMIT })
+        .withMessage('Claps must not exceed 100'),
```

Comment:
Would you consider adding a different validation to check if the clap parameter is an integer (or it's empty as well). 
With the current implementation, users will get the message '_Claps must not exceed 100_' if they enter

- An invalid number
- The clap parameter is not supplied